cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -52,14 +52,47 @@ const (
 	kubeScheduler         = "kube-scheduler"
 )
 
-// WriteStaticPodManifests builds manifest objects based on user provided configuration and then dumps it to disk
-// where kubelet will pick and schedule them.
-func WriteStaticPodManifests(cfg *kubeadmapi.MasterConfiguration) error {
+// CreateInitStaticPodManifestFiles will create and write to disk all static pod manifest files necessary in the kubeadm init phase
+// to establish the control plane, including eventually static pod manifest if necessary.
```

Comment:
I'm not quite sure what is meant by that last clause. Perhaps "CreateInitStaticPodManifestFiles will write all static pod manifest files needed to bring up the control plane."